pwndbg/commands/attachp.py

```diff
@@ -82,6 +82,24 @@ def attachp(no_truncate, target) -> None:
             except CalledProcessError:
                 pids = []
 
+            if not pids:
+                try:
+                    command = "ps aux | awk '{print $2, $11}'"
```

Comment:
Ideally, we don't want to rely on `ask` program existence on the user system. Can we just invoke `["ps", "aux"]` and then filter out the result in Python code?

Reply:
Hi @disconnect3d ,
How about using `ps -eo pid,args` instead?

```bash
root ➜ /pwndbg $ ps aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.0   7632  4296 pts/0    Ss   01:52   0:00 /bin/bash
root     10450  0.0  0.0   7632  4408 pts/1    Ss   06:29   0:00 bash
root     10655  0.0  0.0   2772   976 pts/1    S+   06:30   0:00 ./example-system-x86_64
root     10875  0.0  0.0  10068  1564 pts/0    R+   06:44   0:00 ps aux

root ➜ /pwndbg $ ps -eo pid,args
  PID COMMAND
    1 /bin/bash
10450 bash
10655 ./example-system-x86_64
10885 ps -eo pid,args
```

For example, with PID 10885, I would only capture `ps`, not `ps -eo pid,args`. 
Do you think this approach is acceptable?

Reply:
Using`ps` is fine. I think capturing and matching the args is also fine in general: users may like this too.